README.md

```diff
@@ -74,6 +74,13 @@ There are canonical examples defined in the class definition Javadoc of the clie
 Documentation can be found at the [Java Client
 Github Project Page](http://prometheus.github.io/client_java).
 
+### Disabling `_created` metrics
+
+By default, counters, histograms, and summaries export an additional series
+suffixed with `_created` and a value of the unix timestamp for when the metric
+was created. If this information is not helpful, it can be disabled by setting
+the environment variable `PROMETHEUS_DISABLE_CREATED_SERIES=True`.
```

Comment:
I guess Python's `True` has a capital `T`, but as this is Java, I'd use `true` with a lower case `t` in the documentation to avoid confusion. The implementation is case-insensitive anyway, which is good, but i think people are more used to `true`.

Reply:
For the record the code will accept any variation of `true` - even `tRuE`.

Reply:
Yes, having a case insensitive implementation (and an implementation compatible with Python's) is good.